common/views/components/DefaultPageLayout/DefaultPageLayout.js

```diff
@@ -249,6 +250,7 @@ const DefaultPageLayout = ({
       <div id='main' className='main' role='main'>
         {children}
       </div>
+      <Footer openingHoursId='footer' extraClasses='' />
```

Comment:
Why the `emptyString`?

Reply:
Good point – I think it must have been my lazy way of getting Flow to pass where `extraClasses` as a required prop for something. Will update.